lib/fortnox/api/types.rb

```diff
@@ -1,6 +1,25 @@
 require 'dry-struct'
 require 'dry-types'
 
+module Dry
+  module Types
```

Comment:
We should definitely add test this override. We want to verify it's working as intended and we also want to see if it breaks when we update `Dry::Types` in the future.

Reply:
We could make this a refinement instead?

Reply:
That's a great suggestion! Then we only need to test that our `Types` is behaving as intended and it will prevent there modifications from leaking too :)

Reply:
Turns out you can't refine modules, only classes. I tried a few workarounds but didn't get it to work at all with refinements :/
I'll probably open a PR against dry types and try to get this into the real repo. If not I'll ask @solnic to advice on how to do this refinement in a safe way ...